app/views/create/fromimage.html

```diff
@@ -365,6 +365,9 @@
                               expand="true"
                               can-toggle="false"
                               >
+                              <div class="learn-more-block">
+                                <a href="" ng-click="showComputeUnitsHelp()">What are millicores and MiB?</a>
+                              </div>
```

Comment:
@jwforres Different text here because there is already an "About Resource Limits" link for the form section. Let me know if you have a better idea for where to put it.

![openshift_web_console](https://cloud.githubusercontent.com/assets/1167259/21117489/ea7484aa-c087-11e6-871c-a066070aae2a.png)